### DISCUSSION TITLE
Python not recognized in repo

### DISCUSSION THREAD

**Discussion by ds2606** [github-linguist/linguist#5865] [Classification]
## Describe the bug

My repo backend ([ds2606/chessli](https://github.com/ds2606/chessli)) is all python, and the language breakdown shows all javascript because the file `bootstrap.bundle.js` is being included by linguist (outweighing everything else)

### Expected behaviour

Do not include vendored `bootstrap.bundle.js` file. I expected about 50% python 50% js

```
97.00%  224666     JavaScript
1.10%   2538       HTML
0.82%   1896       Python
0.70%   1625       CSS
0.37%   857        Shell
0.01%   26         Procfile

Procfile:
Procfile

Shell:
chessli.bash

Python:
chessli.py
chessli_lib.py

JavaScript:
static/bootstrap-4.3.1-dist/js/bootstrap.bundle.js
static/controller.js

HTML:
static/index.html

CSS:
static/styles.css
```

**Comment by lildude** [comment]
Whoops. I thought this was a simple classification issue but now I see where the issue really is: the regex used for marking bootstrap files as vendored isn't catching this one file. I'll open an issue again.

In the meantime, you can solve this by implementing an override to mark the file as vendored.

**Reply by lildude** [reply]
https://github.com/github/linguist/issues/5866

**Comment by ds2606** [comment]
Thanks, everything's fixed for me now! I'll watch #5866 to see when I can remove the `.gitattributes` entry
